File: problemas/iniciante/1142/1142.cs
using System;

class URI {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine());
        for(int i = 1; i < 4 * N; i += 4){
            Console.WriteLine($"{i} {i + 1} {i + 2} PUM");
        }
    }
}

File: problemas/iniciante/1142/1142.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  for (int i = 1; i < 4 * n; i += 4) {
    print('${i} ${i + 1} ${i + 2} PUM');
  }
}
